.appveyor.yml

```diff
@@ -11,10 +11,28 @@ configuration:
   - Release
 
 before_build:
-  - git clone --depth=1 https://github.com/KhronosGroup/OpenCL-Headers inc/OpenCL-Headers
-  - cmake -DOPENCL_ICD_LOADER_HEADERS_DIR=inc/OpenCL-Headers -H. -Bbuild -A%PLATFORM%
+  - git clone https://github.com/MathiasMagnus/OpenCL-Headers --branch pkgconfig --single-branch
+  - cd OpenCL-Headers
+  - mkdir build
+  - mkdir install
+  - cd build
+  - cmake -A %PLATFORM%
+          -D CMAKE_INSTALL_PREFIX=../install
+          ..
+  - cmake --build .
+          --target INSTALL
+  - cd ..\..\
+  - git clone https://github.com/MathiasMagnus/OpenCL-ICD-Loader --branch pkgconfig --single-branch
```

Comment:
Should also point to the Khronos repo, master branch.

Reply:
I skimmed over appveyor still being there. The entire file needs to be deleted as GA has all the types of runners we covered with appveyor. (AFAIK, but do correct me if I'm wrong.)